modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -8125,6 +8125,413 @@
           "required": true
         }
       }
+    },
+    "/veterans/{veteranId}/power-of-attorney": {
+      "get": {
+        "summary": "Find current Power of Attorney for a Veteran.",
+        "tags": [
+          "Power of Attorney"
+        ],
+        "operationId": "findPowerOfAttorney",
+        "security": [
+          {
+            "productionOauth": [
+              "system/claim.read",
+              "system/system/claim.write"
+            ]
+          },
+          {
+            "sandboxOauth": [
+              "system/claim.read",
+              "system/system/claim.write"
+            ]
+          },
+          {
+            "bearer_token": [
+
+            ]
+          }
+        ],
+        "description": "Retrieves current Power of Attorney for Veteran.",
+        "parameters": [
+          {
+            "name": "veteranId",
+            "in": "path",
+            "required": true,
+            "description": "ID of Veteran",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response with a current Power of Attorney",
+            "content": {
+              "application/json": {
+                "example": {
+                  "code": "A1Q",
+                  "name": "Firstname Lastname",
+                  "phone": {
+                    "number": "555-555-5555"
+                  },
+                  "type": "individual"
+                },
+                "schema": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "required": [
+                    "code"
+                  ],
+                  "properties": {
```

Comment:
Should all these (not required) fields be marked `nullable`? In local dev, we saw `null`s for those, not sure if that'd happen in prod though.

Reply:
I added those